Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -10,6 +10,45 @@ import Foundation
 import GoogleAPIClientForREST
 import GTMSessionFetcher
 
+struct CategoryRow: Hashable {
+  let categoryName: String
+  let available: String
+  let spent: String
+  let budgeted: String
+  
+  init(row: [String]) {
+    categoryName = row[0]
+    available = row[1]
+    spent = row[4]
+    budgeted = row[7]
+  }
+}
+
+struct GroupsAndCategories {
+  let groups: [String]
+  let groupedCategoryRows: [[CategoryRow]]
+  
+  init(rows:[[String]]) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/b2681273972105974e9d804cc974a7381160826c/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L31"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>GoogleSheetsManager.swift:31</code></td>
    </tr>
  </tbody>
</table>